CHANGE_LOG.md

```diff
@@ -104,3 +104,4 @@ All notable changes to this project will be documented in this file.## [Unreleas
 - Fixed Postchat loading twice issue
 - Adding an event listener in case the chat window is hidden for notifications
 - Adding support for bot survey
+- Fixing reconnect pane not appearing
```

Comment:
We might need to add more details to issues fixed in this PR

Fixing reconnect pane is not appearing in chat widget even when reconnect is configured.
Fixing customer conversation forcefully ended after reconnect time due to first issue.

Reply:
Updated a bit, we definitely don't want to keep the change.log to have PR descriptions.